### PR TITLE
NN-1657 Added dropdown arrow to menu

### PR DESCRIPTION
**Pull request by jsrobertson**
- Swapped from scss to styled
- Added dropdown arrow to menu which rotates when open

### CODE REVIEW

File: components/Dropdown/dropdown.scss [hunk 1]
```diff
@@ -1,111 +0,0 @@
-@import "colours/palette";
```
Comment (petergphillips) [inline components/Dropdown/dropdown.scss]: any reason for leaving the empty file here?

File: components/Dropdown/index.js [hunk 2]
```diff
@@ -1,103 +1,3 @@
...
-  user: {
-    firstName: 'first',
-    activeCaseLoadId: 'id',
-    isOpen: false,
-  },
-  menuOpen: false,
-  extraLinks: [],
-  caseChangeRedirect: true,
-}
+import Dropdown from './Dropdown'
```
Comment (petergphillips) [inline components/Dropdown/index.js]: why add this layer of indirection?

File: components/Dropdown/dropdown.scss (hunk 1 above)
Comment (jsrobertson) [reply components/Dropdown/dropdown.scss]: It's used in header.scss at the moment.

File: components/Dropdown/index.js (hunk 2 above)
Comment (jsrobertson) [reply components/Dropdown/index.js]: Good question.

Not something we've really discussed in fairness, just something I've got into the habit of doing.  Open to the discussion if there's any other preferences/ideas though...

1) Avoids `import Dropdown from '../components/Dropdown/Dropdown`
2) Avoids having a million index.js files/tabs open in your editor and not knowing what's what.

So a component file structure (only in my opinion, of course) should be:

/ComponentName
- index.js
- ComponentName.js
- ComponentName.styles.js (if the styles bloat up ComponentName.js)
- ComponentName.test.js

File: components/Dropdown/dropdown.scss (hunk 1 above)
Comment (petergphillips) [reply components/Dropdown/dropdown.scss]: sorry, didn't make myself clear.  You've cleared the contents of dropdown.css, but left the file in place.  I'm suggesting removing the file instead.

**Comment by petergphillips** [comment]
LGTM.  Don't know what's going on with this project, I can't approve via github :man_shrugging:

File: components/Dropdown/dropdown.scss (hunk 1 above)
Comment (jsrobertson) [reply components/Dropdown/dropdown.scss]: Silly me.  Yeah, no reason at all.  Will remove, cheers.

**Pull request merged**